_emails/us/pennsylvania/statecollege.md

```diff
@@ -0,0 +1,28 @@
+---
+title: State College, Pennsylvania
+permalink: "/statecollegepa"
+name: Letter to Council Members and The Mayor
+city: State College
+state: PA
+recipients:
+  - mayor@statecollegepa.us
+  - boro@statecollegepa.us
+body: |
+  Dear Mayor and Borough Council,
+
+  My name is [YOUR NAME] and I am a resident of [NEIGHBORHOOD/BOROUGH/CITY]. I am writing to ask you to defund the State College Police department.
+
+  In 2019 Osaze Osagie was murdered by a State College Police Officer. Osaze's murder was deemed "justified" by the DA, but no citizen should be killed by their city. This use of public funds to kill is unacceptable.
+
+  According to the 2020 budget, police occupy the majority of all city general fund expenditures. At 38.3%, this over shadows every other public project. The funding used on the police department must be reallocated to public programs which help people, such as Osaze who was in a state of mental distress, rather than on the use of force to end lives.

```

Comment:
```suggestion
  According to the 2020 budget, police occupy the majority of all city general fund expenditures. At 38.3%, this over shadows every other public project. The funding used on the police department must be reallocated to useful social programs which help people, such as Osaze who was in a state of mental distress, rather than on the use of force to end lives. We must invest in housing, jobs, youth programs, restorative justice, and mental health workers to keep the community safe

```